mteb/tasks/PairClassification/multilingual/XNLI.py

```diff
@@ -103,19 +103,22 @@ def dataset_transform(self):
     "odiya": ["ory-Orya"],
     "sanskrit": ["san-Deva"],
     "tamil": ["tam-Taml"],
+    "turkish": ["tur-Latn"],
+    "greek": ["ell-Grek"],
+    "russian": ["rus-Cyrl"],
 }
 
 
-class XNLI2(XNLI):
+class XNLIV2(MultilingualTask, AbsTaskPairClassification):
     metadata = TaskMetadata(
-        name="XNLI2",
+        name="XNLIV2",
         dataset={
-            "path": "mteb/xnli2.0-multi-pair",
-            "revision": "1a3794e88c74db2a13aebc5e8abc3dc4312a7543",
+            "path": "mteb/xnli2.0-multi-pair-13",
```

Comment:
Sould we remove the old one then https://huggingface.co/datasets/mteb/xnli2.0-multi-pair ? I thought you'd just update `xnli2.0-multi-pair`

Reply:
it didn't allow me, then I made a new one that includes the current languages. I tried to delete the older one but it did not allow me. Could you please try to delete the older one?

Reply:
You couldn't open a PR on the repo? I don't have my laptop with me atm, I can do it tonight

Reply:
From my attempts on my own datasets, I haven't found the way to delete through a PR. Probably needs to happen from the interface with a person who has the right access? (I guess I am a member but that might not be enough)

Let me know if u have another idea

Reply:
@Andrian0s I jus opened a PR here: https://huggingface.co/datasets/mteb/xnli2.0-multi-pair/discussions/3
I'll merge it and you'll just need to update the path in this PR to use this repository.